packages/trpc/server/routers/viewer/eventTypes.tsx

```diff
@@ -296,6 +298,13 @@ export const eventTypesRouter = createProtectedRouter()
         data.customInputs = handleCustomInputs(customInputs, id);
       }
 
+      if (bookingLimits) {
+        const isValid = validateBookingLimitOrder(bookingLimits);
+        if (!isValid)
+          throw new TRPCError({ code: "BAD_REQUEST", message: "Booking limits must be in ascending order." });
```

Comment:
nit: 
`validateBookingLimitOrder` theoretically can check for any other issues, so it knows best what error should be thrown.